.github/workflows/regression-report.yml

```diff
@@ -0,0 +1,248 @@
+#####################################################################################
+# GitHub Action to generate Checkstyle report.
+#
+# Workflow starts when:
+# 1) issue comment - created
+#
+# Requirements:
+# 1) secrets.AWS_ACCESS_KEY_ID - access key for AWS S3 service user
+# 2) secrets.AWS_SECRET_ACCESS_KEY - security access key for AWS S3 service user
+#
+# If you need to change bucket name or region, change AWS_REGION and AWS_BUCKET_NAME variables.
+# For another bucket, you will need to change the secrets.
+#####################################################################################
+name: regression-report
+env:
+  AWS_REGION: us-east-2
+  AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  USER_LOGIN: ${{ github.event.issue.user.login }}
+
+on:
+  issue_comment:
+    types: [ created, edited ]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+concurrency:
+  # run_id is guaranteed to be unique and present
+  group: ${{ github.run_id }}
+  cancel-in-progress: false
+
+jobs:
+  parse_comment:
+    if: startsWith(github.event.comment.body, 'Github, generate report for')
```

Comment:
or startWith `GitHub, generate report for`

Reply:
add space after `for` to both trigger phrases.

Reply:
and do extra check for it is PR - https://github.com/orgs/community/discussions/25389#discussioncomment-3596273
to be more narrow and better then original workflow.

Reply:
Done